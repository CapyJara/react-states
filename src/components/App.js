import React from 'react';
import Incrementor from './Incementor';

export default function App() {
  return <Incrementor />;
}
