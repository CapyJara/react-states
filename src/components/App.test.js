import React from 'react';
import App from './App';
import { shallow } from 'enzyme';

describe('app', () => {
  it('renders app', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});

