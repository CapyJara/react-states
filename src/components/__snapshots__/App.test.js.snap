// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app renders app 1`] = `<Incrementor />`;
